Source/Core/DolphinQt/FIFO/FIFOPlayerWindow.cpp

```diff
@@ -6,9 +6,13 @@
 
 #include <QCheckBox>
 #include <QDialogButtonBox>
+#include <QEvent>
```

Comment:
Your commit comment.

>This way, it can be focused with the render window behind it

Did you mean?

>This way, it can be focused with the render window in front of it

?

You then go on to say:

>instead of having the main window show up and cover the render window

Which seems opposite to the first comment.

Reply:
Nope, I meant exactly what I said.  Before, you could only have this:

![image](https://user-images.githubusercontent.com/8334194/112709734-0e7a1180-8e79-11eb-9616-431a2d5bbed0.png)

or this:

![image](https://user-images.githubusercontent.com/8334194/112709738-1d60c400-8e79-11eb-88da-384b26547bfb.png)

i.e. the main window is always directly behind the FIFO player; you can't have the FIFO player and then the render window without the main window in the way (it's much more annoying when the main window is in fullscreen; the workaround I used before was to drag the main window into the corner so it's mostly invisible).  Perhaps it's more of a problem for me due to having a fairly small display.

Reply:
Ah thanks.  I see what you mean now and your description makes sense just needed pictures to fully grasp it :)